### PR TITLE
Form builder and email 

### PR DESCRIPTION
**Pull request by eddyparkinson**
Allows simple form creation and email

**Form**
http://nodejs4-cellmaster.rhcloud.com/tbxr3r35wsoj/appeditor
http://nodejs4-cellmaster.rhcloud.com/tbxr3r35wsoj/app

**Email:** 
e.g. http://nodejs4-cellmaster.rhcloud.com/y9schfpdg03z

### CODE REVIEW

**Comment by audreyt** [comment]
This is _awesome_ work — thank you so much! I will (hopefully) get around to code review sometime next week, perhaps with some delays because of the Lunar New Year in Taiwan.

I'd like to invite other collaborators in this repo to try this out and share feedbacks and thoughts with us.

**Comment by eddyparkinson** [comment]
>I'd like to invite other collaborators in this repo to try this out and share feedbacks and thoughts with us.
Please do, feedback is much appreciated. Am looking for early adopters. 

Audrey - Have you ever considered attending Y-Combinator?

**Comment by audreyt** [comment]
Hi, I'm based in Taipei, Taiwan (and Paris, for two months a year) and so am not actively looking for a U.S. venture. But if someone is willing to take EtherCalc further by applying to YC — essentially what Hackpad did based on Etherpad — I'd be more than happy to support them.

**Comment by eddyparkinson** [comment]
Taipei: I was in Taipei a few years ago. I was best man at my friend's wedding there. Small world. 

Code review: 
1) Some of the features are still work in progress. So expect another release or two. 
2) The code review is needed and much appreciated, learning my way around LiveScript and the ethercalc code is still ongoing.

File: socialcalcconstants.js [hunk 1]
```diff
@@ -199,7 +211,7 @@ SocialCalc.Constants = {
 
    //*** SocialCalc.TableEditor
 
-   defaultImagePrefix: "images/sc_", // URL prefix for images (e.g., "/images/sc")
+   defaultImagePrefix: "/images/sc_", // URL prefix for images (e.g., "/images/sc")
```
Comment (audreyt) [inline socialcalcconstants.js]: Some people run EtherCalc not at `/` but relatively — I would prefer that we keep this unless there's practical problems

**Comment by audreyt** [comment]
Everything look quite sensible to me :-) Merged to `master` and will tweak a bit more from here tomorrow.

Thanks again for the contribution!

**Pull request merged**